src/pykoala/corrections/atmospheric_corrections.py

```diff
@@ -135,12 +151,38 @@ def apply(self, spectra_container, airmass=None):
 # Atmospheric Differential Refraction
 # =============================================================================
 
-def get_adr(data_container, max_adr=0.5, pol_deg=2, plot=False):
-    """Computes the ADR for a given DataContainer."""
+def get_adr(spectra_container : SpectraContainer, max_adr=0.5, pol_deg=2,
+            plot=False):
+    """Computes the ADR for a given DataContainer.
+    
+    Notes
+    -----
```

Comment:
I'm not sure whether this is the only place I've noticed it, but Notes 

> An optional section that provides additional information about the code, possibly including a discussion of the algorithm. This section may include mathematical equations, written in LaTeX format

should always go at the end. This section is perhaps the Extended Summary:
> A few sentences giving an extended description. This section should be used to clarify functionality, not to discuss implementation detail or background theory, which should rather be explored in the [Notes](https://numpydoc.readthedocs.io/en/latest/format.html#notes) section below. You may refer to the parameters and the function name, but parameter descriptions still belong in the [Parameters](https://numpydoc.readthedocs.io/en/latest/format.html#params) section.

Reply:
Right. This has been corrected.